### ISSUE TITLE
Display line numbers in code samples

### ISSUE THREAD

**Issue by quassy** [rabbitmq/rabbitmq-website#902]
For example when looking at a description in the [Python Tutorial #6](https://www.rabbitmq.com/tutorials/tutorial-six-python.html) like

> (32) At this point we can sit back and wait until the proper response arrives.

... it's quite hard to follow which code line this refers to, without seperately opening the source code either in GitHub or your own editor. Code blocks (maybe only longer ones) should have line numbers to make it easier to follow the tutorials.

**Comment by michaelklishin** [comment]
highlight.js does not support line numbers [by design](https://highlightjs.readthedocs.io/en/latest/line-numbers.html). FWIW this is not a common request. I'd consider linking to executable (GitHub) repo more. A PR that does that would be considered.
